course_viewer/course_viewer/models.py

```diff
@@ -0,0 +1,44 @@
+from django.db import models
+
+
+class State(models.Model):
+    '''
+    The Transfer State
+    '''
+    state_code = models.CharField(max_length=2,
+                                  help_text='',
+                                  unique=True)
+    state_name = models.CharField(max_length=200,
+                                  unique=True)
+
+
+class College(models.Model):
+    '''
+    The Transfer College
+    '''
+    college_code = models.CharField(max_length=15,
+                                    help_text='',
+                                    unique=True)
+    college_name = models.CharField(max_length=200,
+                                    unique=True)
```

Comment:
College names aren't explicitly unique, only the codes are.  There are some colleges with the exact same name.